lte/gateway/c/session_manager/SessionProxyResponderHandler.cpp

```diff
@@ -108,7 +108,13 @@ void SessionProxyResponderHandlerImpl::AbortSession(
     ServerContext* context, const AbortSessionRequest* request,
     std::function<void(Status, AbortSessionResult)> response_callback) {
   PrintGrpcMessage(static_cast<const google::protobuf::Message&>(*request));
-  const auto imsi       = request->user_name();
+  auto imsi = request->user_name();
+  // SessionD currently stores IMSIs with the 'IMSI' prefix so append if it is
+  // not there already
+  if (imsi.find("IMSI") == std::string::npos) {
+    imsi = "IMSI" + imsi;
+    MLOG(MDEBUG) << "Appended 'IMSI' to ASR user_name: " << imsi;
+  }
```

Comment:
Why don't you that on session proxy using `AddIMSIPrefix` . We do it for the other requests

I think you just need to add that in here

https://github.com/magma/magma/blob/509e5b4bcdb68b0021044fb1dc8cf47554446b8f/feg/gateway/services/session_proxy/credit_control/asr.go#L67